src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
 export interface Figure {
+  color: string;
+  shape: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function isTrianglePossible(a: number, b: number, c:number): boolean {
+  const sum = [a, b, c];
+  const largest = Math.max(a, b, c);
 
+  for (let i = 0; i < 3; i += 1) {
+    if (sum[i] === largest) {
+      sum[i] = 0;
+    }
+  }
+
+  const whatIsLeft = sum.reduce((total, x) => total + x, 0);
```

Comment:
Also, think about another name instead of `whatIsLeft` and I recommend not to use `x, y, ...` (use `cur`, `side` for ex), it's not a math  =)